### PR TITLE
Update RabbitMQ non-durable messaging section

### PR DESCRIPTION
**Pull request by bording**

### CODE REVIEW

File: nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The default behavior of sending messages via RabbitMQ is to wait for the publisher confirmation before proceeding. When non-durable messaging is used, the RabbitMQ transport skips the publisher confirmation step.
+
+This effectively means the blocking call to [WaitForConfirmsOrDie](http://www.rabbitmq.com/javadoc/com/rabbitmq/client/Channel.html#waitForConfirmsOrDie%28long%29) is skipped.
```
Comment (SimonCropp) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: do we care about the overload? or is `e%28long%29` redundant?

Comment (SimonCropp) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: "effectively "?

Comment (bording) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: That was the existing link, I didn't take a look at it.

That is actually linking to the java version of the docs.

Comment (bording) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: Copy and paste from what was already there on the page.

Comment (SimonCropp) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: @bording ok. but isnt it redundant?

Comment (bording) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: I suppose so. My emphasis was on making the sure the content was correct for the current version vs. rewriting everything for the previous version of the content.

Comment (SimonCropp) [inline nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md]: ok fair enough. i will fix those later

**Comment by bording** [comment]
@SimonCropp Just pushed up a change to make the v5 info line up with my new v6 changes.

**Pull request merged**